src/pybind/mgr/volumes/fs/nfs.py

```diff
@@ -395,6 +383,27 @@ def _check_rados_notify(ioctx, obj):
         except TimedOut:
             log.exception(f"Ganesha timed out")
 
+    @property
+    def clusters(self):
+        return available_clusters(self.mgr)
+
+    @property
+    def exports(self):
```

Comment:
Sorry @batrick removing this commit. Export rados objects needs to be parsed only when FSExport is initialized, not otherwise.